index.js

```diff
@@ -82,7 +91,12 @@ exports.register = (server, pluginOptions, next) => {
     routeOptions.handler = options.routeHandler ? options.routeHandler : (request, reply) => {
       // the report will be a Buffer representing a JSON string:
       if (request.payload) {
-        server.log(options.logTags, request.payload.toString('utf-8'));
+        try {
```

Comment:
lets do a try catch around JSON.parse and if it fails, just do a payload.toString and then send either of those to server.log